airbyte-metrics/lib/src/main/java/io/airbyte/metrics/lib/MetricQueries.java

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright (c) 2021 Airbyte, Inc., all rights reserved.
+ */
+
+package io.airbyte.metrics.lib;
+
+import static io.airbyte.db.instance.configs.jooq.Tables.ACTOR;
+import static io.airbyte.db.instance.configs.jooq.Tables.ACTOR_DEFINITION;
+
+import io.airbyte.db.instance.configs.jooq.enums.ReleaseStage;
+import java.util.List;
+import java.util.UUID;
+import org.jooq.DSLContext;
+
+/**
+ * Keep track of all metric queries.
+ */
+public class MetricQueries {
+
+  public static List<ReleaseStage> jobIdToReleaseStages(final DSLContext ctx, final long jobId) {
+    final var srcRelStageCol = "src_release_stage";
+    final var dstRelStageCol = "dst_release_stage";
+
+    final var query = String.format("""
+                                    select src_def_data.release_stage as %s,
```

Comment:
super nit: I tend to prefer capitalizing sql key words like SELECT, FROM, INNER JOIN, etc when writing raw sql queries like this just as a readability aid, what do you think?

Reply:
no opinion so happy to adopt your style!